src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs

```diff
@@ -641,10 +640,40 @@ private IEnumerable<VfsStatEntry> GetDirectoryResponse(HttpRequest request, File
         protected HttpResponseMessage CreateResponse(HttpStatusCode statusCode, object payload = null)
```

Comment:
Don't we have code paths passing an exception here? Additional handling would be required to ensure that isn't serialized.

Reply:
Yes, all the calls to this method are in this class itself. Example call where we are passing the entire exception - [link](https://github.com/Azure/azure-functions-host/blob/255b7e59538b9ab677d653daf347b3b0afa70dc9/src/WebJobs.Script.WebHost/Management/VirtualFileSystem.cs#L179)

Reply:
exception is optional parameter and it is handled in the CreateResponse method

Reply:
Checking the payload object is exception to remove the details